src/main/java/com/faforever/client/game/PlayerCardTooltipController.java

```diff
@@ -31,12 +31,16 @@ public PlayerCardTooltipController(CountryFlagService countryFlagService, I18n i
   }
 
   public void setPlayer(Player player) {
+    setPlayerWithRating(player, RatingUtil.getRoundedGlobalRating(player));
+  }
+
+  public void setPlayerWithRating(Player player, int rating) {
```

Comment:
I suggest that we remove `setPlayer()` and introduce an enum `RatingMode` with values `EXACT` and `ROUNDED`.
This can then be passed to this method instead of `rating`, reducing the complexity and putting the rating calculation into one place.